Use system configured maximum exec limit instead of ARG_MAX

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>

diff -up findutils-4.2.33/xargs/xargs.c_old findutils-4.2.33/xargs/xargs.c
--- findutils-4.2.33/xargs/xargs.c_old	2008-03-28 14:04:57.000000000 +0100
+++ findutils-4.2.33/xargs/xargs.c	2008-03-28 14:06:54.000000000 +0100
@@ -440,7 +440,7 @@ main (int argc, char **argv)
        * specifies that it shall be at least LINE_MAX.
        */
 #if defined(ARG_MAX)
-      assert(bc_ctl.arg_max <= (ARG_MAX-2048));
+      assert(bc_ctl.arg_max <= bc_ctl.posix_arg_size_max);
 #endif
 #ifdef LINE_MAX
       assert(bc_ctl.arg_max >= LINE_MAX);
